### DISCUSSION TITLE
Even after adding a custom 404.md, the standard 404 page is displayed.

### DISCUSSION THREAD

**Discussion by kawamataryo** [vuejs/vitepress#3880] [Q&A] [answered]
I am creating documentation using VitePress. To display a custom 404 page, I added a 404.md file. However, after building the project, the standard 404 page is displayed instead. 

Here is the reproduction environment: https://stackblitz.com/edit/vite-pukdjm?file=docs%2F404.md

Accessing ﻿`/404.html` displays the custom 404 page. However, when accessing non-existent paths such as `﻿/foo.html`, the standard 404 page appears instead (the custom 404 page is briefly displayed, but then seems to get replaced).
How can I ensure that the custom 404 page is displayed when accessing non-existent paths?

**Comment by brc-dd** [comment, marked as answer]
Ah yeah. It's broken currently. You can use `not-found` layout slot from theme instead - https://vitepress.dev/guide/extending-default-theme#layout-slots

**Reply by kawamataryo** [reply]
Thank you for the quick response! Is this issue planned to be fixed in the next version? I hope it is resolved soon. 
Additionally, if it has not yet been fixed, I would like to contribute.

**Reply by brc-dd** [reply]
Can be marked as a feature request currently. The code around 404 page is quite fragile and might not be easy to contribute. But yeah if you think you can contribute that's most welcome!

**Reply by otabekoff** [reply]
Hi @brc-dd any updates on this feature?

**Reply by Dollique** [reply]
Is there any way to currently use one of my markdown pages as the "notFound" page? I only managed to hardcode the error page in my component, but I want to use one of my markdown pages. Using a 404.md did not work. Something like a `Layout: 404` property would actually be even better.
